File: app/src/main/java/com/artmurka/artmurkaapp/android/views/activities/IMainActivity.java
package com.artmurka.artmurkaapp.android.views.activities;


import com.artmurka.artmurkaapp.model.pojo.itemlist.categories.Success;

import java.util.List;

public interface IMainActivity {
    void changeFragment(int currentFragment, String url,List<Success> childs, String catName);
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/activities/main/IMainActivity.kt
package com.artmurka.artmurkaapp.android.views.activities.main


import android.os.Bundle
import com.artmurka.artmurkaapp.data.model.pojo.itemlist.categories.Success
import com.artmurka.artmurkaapp.other.FragmentType
import com.artmurka.artmurkaapp.presenter.PresenterView

interface IMainActivity : PresenterView {
    fun changeFragment(fragment: FragmentType, url: String? = null, childs: List<Success>? = null, catName: String? = null, bndl: Bundle? = null)
    fun showToast(msg: String)
    fun exit()
    fun makeCall()


}
